.github/actions/e2e-pw/run-log-tests/action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Run Tests'
+description: 'Runs Playwright E2E tests'
+
+runs:
+    using: 'composite'
+    steps:
+        - name: First Run Playwright E2E Tests
+          id: first_run_e2e_pw_tests
+          # Use +e to trap errors when running E2E tests.
+          shell: /bin/bash +e {0}
+          run: npm run test:e2e-pw
+        - uses: actions/upload-artifact@v3
+          if: always()
+          with:
+              name: playwright-report
+              path: playwright-report/
+              retention-days: 30
```

Comment:
Seems excessive; can probably be kept for 7 days max

Reply:
Thanks @jbordonado! Your feedback was hidden until I marked it as "Ready for review" – I'll make your recommended changes before merging into the parent feature branch #7671.

Reply:
Reduced to 7 days in d98d705